### DISCUSSION TITLE
Problem with more than 15 references

### DISCUSSION THREAD

**Discussion by choxos** [quarto-dev/quarto-cli#503] [Q&A]
Hi,

I am using Quarto for writing a scientific manuscript. We encountered a strange problem. It seems Quarto cannot read more than 15 references and removes every sentence after the 15th reference that contains a reference. How can I fix that?

Ahmad

**Comment by jjallaire** [comment]
If you could provide a minimal reproducible example we should be able to get to the bottom of this quickly!

**Reply by choxos** [reply]
Trying to make a reproducible version, just fixed it! There was a backslash \ before one of the citations. That was because of using special letter (such as ş or é) in author names. Quarto should not add backslash automatically before citations.

**Reply by jjallaire** [reply]
Could you share exactly what the citation key was that caused the problem so we can reproduce locally? Also, I'm assuming by the fact that a backslash was injected that you are using the visual editor in RStudio?

**Reply by choxos** [reply]
The exact file is here: 
https://github.com/raittioe/dental-transparency/blob/main/manuscript_dental_transparency.qmd

The end of discussion before limitations. The rendered html file is also here:
https://github.com/raittioe/dental-transparency/blob/main/manuscript_dental_transparency.html

And yes, I'm using the visual editor.

**Comment by jjallaire** [comment]
Thanks for reporting this! I am working on a fix now. In the meantime you can prevent the visual editor from doing this by adding the following to your document YAML options:

```yaml
---
title: "Transparency of research articles published in dental journals"
from: markdown+tex_math_single_backslash
---
```

**Comment by jjallaire** [comment]
Okay, the various fixes discussed here do address this issue: https://github.com/rstudio/rstudio/issues/10075#issuecomment-1086698733

Hopefully we'll get these into a patch release very soon, in the meantime if you use that workaround you should be okay.

**Reply by choxos** [reply]
Thanks a lot!
